### ISSUE TITLE
Expose joint damping_ratio and natural_frequency per joint to make joints soft

### ISSUE THREAD

**Issue by Ughuuu** [appsinacup/godot-rapier-physics#231]
There have been 2 issues about PinJoint2D not having softness like Godot has. Rapier just doesn't support per joint softness, so as a workaround you can change the following project wide properties:
- `physics/rapier/joint/damping_ratio`
- `physics/rapier/joint/natural_frequency`

These will help to make soft joints (eg damping ratio 0.1 and natural_frequency 50).

However, there isn't a way right now in rapier to set these per joint. Here is linked issue in Rapier https://github.com/dimforge/rapier/issues/676

Also, in order to expose new properties (on the joint directly) and hide the softness, this one needs to be solved first: https://github.com/godotengine/godot-proposals/issues/10438#issuecomment-2307782398

**Comment by GeminiSquishGames** [comment]
> (eg damping ratio 0.1 and natural_frequency 50).

I noticed the range sliders only go from 0 to 10, but started at some huge amount. I guess enter by hand is the key.

**Comment by Ughuuu** [comment]
I can update the range sliders limit, maybe I set it from 0 to 10 by copy pasting other properties.

**Comment by GeminiSquishGames** [comment]
Yeah, I'm using mass quite a bit and noticed I have to use rather lower values. Physics toy, high progression potential, Roguelike, Solar Jetman space shooter type thing.

**Comment by Ughuuu** [comment]
The units are 100, that is how many pixels equal 1 meter. It affects some things, unsure if it affects linear velocity though.

**Comment by GeminiSquishGames** [comment]
I will test things and scale as needed. I'll probably have to scale up things a bit, if the mass is accurately in kilograms then things should be okay as long as I can stick to the scale, but the lack of z dimension probably leaves room for arbitrary approximations. Damage on impact is calculated on velocity. As is it's pixels per second, or if I do it by position each frame manually, pixels per frame. I was going for 128 p/meter so not far off, just to stick with general texture standard powers of 2.
